boards/nxp/mimxrt1064_evk/mimxrt1064_evk.dts

```diff
@@ -254,16 +242,8 @@ zephyr_udc0: &usb1 {
 };
 
 &csi {
-	status = "okay";
-	sdev = <&mt9m114>;
 	pinctrl-0 = <&pinmux_csi>;
 	pinctrl-names = "default";
-
-	port {
-		csi_ep_in: endpoint {
-			remote-endpoint = <&mt9m114_ep_out>;
-		};
-	};
```

Comment:
Do we need to add the labels related to the new shield you have created for 1064?